assets/js/mbta.ts

```diff
@@ -2761,13 +2806,27 @@ const stationConfig: {
           },
         },
       },
+      {
+        id: 'SLEC',
+        name: 'Lechmere - GLX',
+        zones: {
+          m: {
+            modes: {
+              auto: false,
+              custom: true,
+              headway: false,
+              off: true,
+            },
+          },
+        },
+      },
       {
         id: 'GLEC',
```

Comment:
Will definitely double check with Andrew on that. I figured that the `GLEC` zones belonged to the new train stop and the `SLEC` zone belonged to bus, but I wasn't sure if we wanted to still keep the old busway signs considering there are currently still buses running out of the old Lechmere station.

I guess that mainly begs the question of what exactly the rollout plan is for the new zones. Although these changes don't need to go to prod until the station officially opens, I wonder if, when that happens, it will be an immediate and full cutover to the new train station/busway or would there be any reason to continue running buses out of old lechmere for some time when the new station opens?

Reply:
Oh, I was under the impression the PA/ESS signs at the old Lechmere were offline. Can you check with Dan Salomon about that? And I think the new signs are up and running. If that's the case, I think you can do a hard cutover now, and we can just set them to "Off".

Reply:
Ah I see, I spoke to Dan and the signs are still powered on but because they're cut off from the network, there's nothing syncing them to keep them from drifting so I think the times are off by several hours. So it does seem like it'll be fine to completely remove them. Probably also will be worth reaching out to New Era Tech to have those signs turned off since they're not doing anything helpful.